mir_eval/separation.py

```diff
@@ -359,9 +485,22 @@ def evaluate(reference_sources, estimated_sources, **kwargs):
     # Compute all the metrics
     scores = collections.OrderedDict()
 
-    sdr, sir, sar, perm = util.filter_kwargs(bss_eval_sources,
-                                             reference_sources,
-                                             estimated_sources, **kwargs)
+    if win is not None and hop is not None:
```

Comment:
Good point! That would be a user mistake and I'll throw a descriptive error message.